vivisect/__init__.py

```diff
@@ -1287,6 +1306,11 @@ def makeCode(self, va, arch=envi.ARCH_DEFAULT):
 
         logger.debug("makeCode(0x%x, 0x%x)", va, arch)
         calls_from = self.cfctx.addCodeFlow(va, arch=arch)
+        if fva is None:
```

Comment:
we should actually discuss a `getFunctions()` call that returns all the functions an opcode or codeblock belongs to.

Reply:
We should. It'd probably be a breaking change unless it was a new function, but that would be good.